packages/devtools_app/web/debugger_screen.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
```

Comment:
For the code shared between this page and index.html, maybe add a comment to both that updating shared code in one file requires updating the other.

Reply:
Good idea, done.